<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1751</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2D">
     <attribute name="title">
      <string>2D</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>861</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Annotation</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Instance Lists</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Detection List:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>831</width>
         <height>231</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>x1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>x2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>label</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prob</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>INSTANCE</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget_instance">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>281</width>
         <height>131</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>label</string>
        </property>
       </column>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>10</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Automatic Data Association</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>901</width>
        <height>561</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid red</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Loading</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Image Dir:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Detection Dir:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>160</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_imagePath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/media/tongyuan/disk_ubuntu/Ubuntu/dataset/ICL-NUIM/living_room_traj0n_frei_png/rgb/</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_detection_dir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/media/tongyuan/disk_ubuntu/Ubuntu/dataset/ICL-NUIM/living_room_traj0n_frei_png/bbox_testannotation/</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0 / 0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_labelConfig">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/home/tongyuan/workspace/object-slam/annotation/semantic-data-association-tool/data/coco.names</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Label config Path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_instancePath">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/home/tongyuan/workspace/object-slam/datas/ICL-NUIM-annotation/object_list.txt</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>100</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Instance Path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_datasetDir">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>60</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/media/tongyuan/disk_ubuntu/Ubuntu/dataset/ICL-NUIM/living_room_traj0n_frei_png/</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>40</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dataset Dir:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_jump">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Jump">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Jump</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>570</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_left">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_right">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>600</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1350</x>
        <y>600</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3D">
     <attribute name="title">
      <string>3D</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Loading</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pcd Map path:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pcdPath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/media/tongyuan/disk_ubuntu/Ubuntu/dataset/ICL-NUIM/map/map.pcd</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0 / 0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_settingPath">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>60</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/home/tongyuan/workspace/object-slam/Object-oriented-SLAM/Object-oriented-SLAM/Example/param/ICL-NUIM.yaml</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Visualization Setting path:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Instance Path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_instancePath_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>170</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/home/tongyuan/workspace/object-slam/datas/ICL-NUIM-annotation/object_list.txt</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Label config Path:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_labelConfig_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>/home/tongyuan/workspace/object-slam/annotation/semantic-data-association-tool/data/coco.names</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ground Plane:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_groundPlane">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>-0.00162817 0.999984 -0.00542226 1.37843</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Instance Lists</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_instance3D">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>1191</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>x</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>z</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>r</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>p</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>a</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>b</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>c</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>label</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ExtraYaw</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>650</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="pageStep">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>650</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_showtraj">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>110</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Trajectory</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1758</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
